### PR TITLE
Documentation skeleton

### PR DESCRIPTION
**Pull request by darrylmelander**
This is a start on the documentation. What's been completed so far:
* Automatically publish to egret.readthedocs.io
* Tweaked default layout to my liking
* Created the documentation outline
* Wrote a few sections:
   * Installation
   * How to run
   * Configuration Options

I think I'd like to merge what we've got so far, partly to encourage any new development to write documentation as new capabilities are added.  We can continue to fill in documentation in new PRs.

Partially addresses #35

### CODE REVIEW

File: doc/OnlineDocs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Prescient'
+copyright = '2021, Prescient Developers'
+author = 'Prescient Developers'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'
```
Comment (bknueven) [inline doc/OnlineDocs/source/conf.py:25]: We should probably call this `2.0`, to match the [current version in setup.py](https://github.com/grid-parity-exchange/Prescient/blob/530a5d7a72e9b9281c11d32ce1ce3d58653e7e87/setup.py#L26).
```suggestion
release = '2.0'

```

File: doc/OnlineDocs/source/tasks/install.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+If you are using something other than Anaconda to manage your python environment, use the 
+information in `environment.yml` to identify which packages to install. 
+
+
+Install a linear solver
+-----------------------
+Prescient requires a mixed-integer linear programming (MILP) solver that is compatible with
+`Pyomo <https://pyomo.readthedocs.io>`_. Options include open source solvers such as CBC, GLPK,
+or SCIP, and commercial solvers such as CPLEX, Gurobi, or Xpress.
```
Comment (bknueven) [inline doc/OnlineDocs/source/tasks/install.rst:52]: I don't think we actually support SCIP..

**Review by bknueven — APPROVED** [review]
Looks good. I think we should change the version number before merging.

File: doc/OnlineDocs/source/conf.py (hunk 1 above)
Comment (darrylmelander) [reply doc/OnlineDocs/source/conf.py:25]: I noticed on the github project page it lists the latest version as 2.0.1. So bump to 2.0.2?

Comment (bknueven) [reply doc/OnlineDocs/source/conf.py:25]: That would probably be best -- I would just like this to be in lockstep with whatever is the `version` in setup.py.

File: doc/OnlineDocs/source/tasks/install.rst (hunk 2 above)
Comment (darrylmelander) [reply doc/OnlineDocs/source/tasks/install.rst:52]: It's listed in the README.md. I'm fine taking it out.

File: doc/OnlineDocs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Prescient'
+copyright = '2021, Prescient Developers'
```
Comment (darrylmelander) [inline doc/OnlineDocs/source/conf.py:21]: Ought to update the copyright to use the standard NTESS wording, like we have at the top of source code.

**Pull request merged**